pkg/pipeline/extract/aggregate/aggregate.go

```diff
@@ -213,3 +229,21 @@ func (aggregate Aggregate) GetMetrics() []config.GenericMap {
 
 	return metrics
 }
+
+func (aggregate Aggregate) cleanupExpiredEntries() {
+	nowInSecs := time.Now().Unix()
```

Comment:
IMO comparing `Time` objects rather than ints of seconds from Epoch time, is more idiomatic to Go.

Reply:
I added this as a point to issue https://github.com/netobserv/flowlogs-pipeline/issues/181 ... this is currently the same behavior we have in all other places. I keep this as-is for this PR and we will fix it when moving to generic LRU with expiration cache.